packages/sources/coinlore/test/integration/adapter.test.ts

```diff
@@ -0,0 +1,54 @@
+import { AdapterRequest } from '@chainlink/types'
+import request from 'supertest'
+import * as process from 'process'
+import { server as startServer } from '../../src'
+import * as nock from 'nock'
+import * as http from 'http'
+import { mockResponseSuccess } from './fixtures'
+
+describe('execute', () => {
+  const id = '1'
+  let server: http.Server
+  const req = request('localhost:8080')
+  beforeAll(async () => {
+    process.env.CACHE_ENABLED = 'false'
```

Comment:
With the CI integration test fixed, can we make this test without cache disabled?

Reply:
Same goes for other integration test PRs